SDK/include/NDK/Widgets.hpp

```diff
@@ -1,4 +1,4 @@
-// This file was automatically generated
+﻿// This file was automatically generated
```

Comment:
Please disable UTF-8 BOM in Visual Studio https://vlasovstudio.com/fix-file-encoding/